suite-native/assets/src/components/DiscoveryAssetsLoader.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+import { ListItemSkeleton, HStack, Text } from '@suite-native/atoms';
+import { Icon } from '@suite-common/icons';
+import { useTranslate } from '@suite-native/intl';
+
+type DiscoveryAssetsLoaderProps = {
+    isListEmpty: boolean;
+};
+
+export const DiscoveryAssetsLoader = ({ isListEmpty }: DiscoveryAssetsLoaderProps) => {
+    const { translate } = useTranslate();
+
+    const discoveryProgressText = translate(
+        isListEmpty
+            ? 'assets.dashboard.discoveryProgress.1'
+            : 'assets.dashboard.discoveryProgress.2',
+    );
+
+    const numberOfSkeletons = isListEmpty ? 3 : 1;
```

Comment:
The number 3 could be a constant also used in `Assets.tsx` since they kind of rely on each other

Reply:
Good idea, but defining this const in `Assets.tsx` would lead to cyclic dependency and placing it here and exporting it to `Assets` doesn't feel right to me.

Reply:
I've at least created the const on top of the `Assets` file. ab21592d4

Reply:
There is couple of options: 
- create constants file to get rid of dependencies
- import from this file to Assets - nothing wrong with that. Makes sense why it's done since you're importing the component there as well

Reply:
constants created in https://github.com/trezor/trezor-suite/commit/019dcf358b6a327c6c54804d511837ec2b74aa82